src/jit/flowgraph.cpp

```diff
@@ -15954,6 +15949,15 @@ REPEAT:;
         fgDispBasicBlocks(verboseTrees);
         fgDispHandlerTab();
     }
+
+    if (compRationalIRForm)
+    {
+        for (BasicBlock* block = fgFirstBB; block != nullptr; block = block->bbNext)
+        {
+            assert(LIR::AsRange(block).CheckLIR(this));
+        }
```

Comment:
It seems weird to assert(...CheckLIR) since that function will assert if there is any problem. Shouldn't you just call CheckLIR()?